lib/languages/raku.txt

```diff
@@ -0,0 +1,10 @@
+!e
+#`(+\j-->>)+\pj
+#`{+\j-->>}+\pj
+#`[+\j-->>]+\pj
+#`(\j-->>)\pj
+#`{\j-->>}\pj
+#`[\j-->>]\pj
+#
+=begin-->>=end
+unit
```

Comment:
I would probably exclude the `unit` declaration from this, as PHP seems to keep their class declarations intact (plus it means we don't have to deal with `unit class Foo;` vs `class Foo {}` behaving differently): https://github.com/exercism/snippet-extractor/blob/main/test/languages/php_test.rb#L40

Reply:
I was trying to get rid of `unit module` which appears unnecessary in raku solutions (eg Luhn). Do you think we should adjust `unit` to `unit module` or still just drop it?